### ISSUE TITLE
deck.gl Multiple Layers chart broke after 1.2.0

### ISSUE THREAD

**Issue by zgramana** [apache/superset#16572]
deck.gl Multiple Layers chart is broken, causing the Explore page to fail to render

### Expected results

Open a deck.gl Multiple Layer chart see a map rendered

### Actual results

The entire page below the menu bar disappears as soon as the "Run Query" button is opened

#### Screenshots

<img width="1286" alt="image" src="https://user-images.githubusercontent.com/759830/131915470-4a53e9a2-2c5b-48bb-8dfa-89fc63134bf0.png">

#### How to reproduce the bug

1. Create a deck.gl Multiple Layer chart,
2. Add "Path" chart as the  layer
3. Press "Run Query"

### Environment

Currently using via `docker-compose -f docker-compose-non-dev.yml up`

- superset version: DockerHub `apache/superset@sha256:c6956897004b8072f97ee8e1a9ea245284b373712c96929ab13ca954164e2348`
- python version: `Python 3.7.9`
- node.js version: `node -v`
- any feature flags active:
```json
FEATURE_FLAGS = {
    "ALERT_REPORTS": True,
    "DASHBOARD_NATIVE_FILTERS": True,
    "DASHBOARD_CROSS_FILTERS": True,
    "DASHBOARD_NATIVE_FILTERS_SET": True,
    "DASHBOARD_FILTERS_EXPERIMENTAL": True,
    "OMNIBAR" : True,
    "TAGGING_SYSTEM" : True,
    "DYNAMIC_PLUGINS" : True,
}
```

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x ] I have reproduced the issue with at least the latest released version of superset.
- [x ] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

Observed errors in the JS console:

<img width="860" alt="image" src="https://user-images.githubusercontent.com/759830/131917514-56500900-7289-462c-83d2-0e031b2a2920.png">

**Comment by rosemarie-chiu** [comment]
@junlincc @evans this issue is reported again today

**Comment by rusackas** [comment]
Resolved by #17703 - let us know if there are more problems remaining that mean we should reopen this issue.
